app/assets/stylesheets/helpers/_header.scss

```diff
@@ -472,76 +472,104 @@
   display: none;
 }
 
-$light-blue: #259EDA;
+$covid-green: #7eff8e;
```

Comment:
Non-blocker, we may be able to re-use [`$gem-c-covid-green` from the gem](https://github.com/alphagov/govuk_publishing_components/blob/a154b2c60b6d8c80cf41e66ee63557353627548e/app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss#L98).

Reply:
@alex-ju That doesn't seem to work - it says `Undefined variable: "$gem-c-covid-green"`

Reply:
Ah, go with a hardcoded value then – it may be that we don't include all those SCSS files at the moment.